ZenPacks/zenoss/Microsoft/Windows/zenpack.yaml

```diff
@@ -448,6 +448,15 @@ classes:
         type: int
       instance_name:
         details_display: false
+      storage_disk_lun:
+        # link to storage disk/virtual drive
+        # if the hard disk is actually mapped to it
+        label: Storage Disk/LUN
+        type: entity
+        grid_display: false
```

Comment:
grid_display is false by default for HardDisk properties.  no need to add it here